content/applications/inventory_and_mrp/inventory/barcode/operations/receipts_deliveries.rst

```diff
@@ -0,0 +1,183 @@
+
+Once the checkbox is ticked, click :guilabel:`Save` at the top of the page to save changes. Once the
+page has refreshed, new options will be displayed under the :guilabel:`Barcode Scanner` feature:
+:guilabel:`Barcode Nomenclature` (with a corresponding drop-down menu), where either
+:guilabel:`Default Nomenclature` or :guilabel:`Default GS1 Nomenclature` can be selected, a
+:guilabel:`Configure Product Barcodes` internal link arrow, and a set of :guilabel:`Print` buttons
+for printing barcode commands and a barcode demo sheet.
+
+.. image:: receipts_deliveries/receipts-deliveries-barcode-setting.png
+   :align: center
+   :alt: Enabled Barcode feature in Inventory app settings.
+
+For more on setting up and configuring the :guilabel:`Barcode` app, refer to the :doc:`Set up your
+barcode scanner </applications/inventory_and_mrp/inventory/barcode/setup/hardware>` and
+:doc:`Activate the Barcodes in Odoo
+</applications/inventory_and_mrp/inventory/barcode/setup/software>` documentation pages.
+
+Scan barcodes for receipts
+==========================
+
+To process warehouse receipts for incoming products, there first needs to be a purchase order (PO)
+created, and a receipt operation to process.
+
+To create a :abbr:`PO (purchase order)`, navigate to the :menuselection:`Purchase app --> Create` to
+create a new request for quotation (RFQ). From the blank :abbr:`RFQ (request for quotation)` form,
+click the drop-down menu next to the :guilabel:`Vendor` field to add a vendor. Then, on the
+:guilabel:`Product` line under the :guilabel:`Products` tab, click :guilabel:`Add a product`, and
+select the desired product(s) to add to the quotation.
+
+Once ready, click :guilabel:`Save` at the top of the form, and click :guilabel:`Confirm Order` to
+confirm the :abbr:`RFQ (request for quotation)` to a :abbr:`PO (purchase order)`.
+
+.. image:: receipts_deliveries/receipts-deliveries-purchase-order.png
+   :align: center
+   :alt: Completed purchase order for barcode product.
+
+To process and scan barcodes for warehouse receipts, navigate to the :menuselection:`Barcode app`.
+Once inside the :guilabel:`Barcode app`, a :guilabel:`Barcode Scanning` screen displaying different
+options is presented. To process receipts, click on the :guilabel:`Operations` button at the bottom
+of the screen. This navigates to an :menuselection:`Operations` overview page.
+
+.. image:: receipts_deliveries/receipts-deliveries-barcode-scanner.png
+   :align: center
+   :alt: Barcode app start screen with scanner.
+
+From this page, locate the :guilabel:`Receipts` card, and click the :guilabel:`# To Process` button
+to view all outstanding receipts. Then, select the desired receipt operation to process. This
+navigates to the barcode transfer screen.
+
+.. note::
+   If using only a barcode scanner or the Odoo mobile app, the barcodes for each transfer
+   of a corresponding operation type can be scanned to be processed easily. Once scanned, the
```

Comment:
I'm slightly confused by what you're trying to communicate in this first sentence. Are you saying it's easier to scan things when Odoo *Barcode* is used alone?

Is this somewhat what you were thinking of saying?
```suggestion
   When Odoo *Barcode* is used as a standalone app, begin by scanning the barcode for each transfer
   type (e.g. receipts, deliveries, picks). Once scanned, the
```